recipe-server/client/control_new/components/tables/DataList.js

```diff
@@ -46,7 +46,12 @@ export default class DataList extends React.Component {
 
   handleChangeSortFilters(pagination, filters, sorter) {
     const { getCurrentURL, push } = this.props;
-    const filterParams = mapObject(filters, values => values.join(',') || undefined);
+    const filterParams = mapObject(filters, values => {
```

Comment:
This could go back to the one-liner with a small change:
`const filterParams = mapObject(filters, values => values && values.join(',') || undefined);`